.github/workflows/main.yml

```diff
@@ -205,6 +205,45 @@ jobs:
       #      -Ddep.iceberg.version=${ICEBERG_VERSION} \
       #      -Ddep.nessie.version=${NESSIE_VERSION}
 
+  iceberg143_nessie0780:
+    name: Nessie 0.78.0 / Iceberg 1.4.3
```

Comment:
i assume we are intentionally adding this as "latest 1.4.x combo" ?

what is currently stopping us from adding a 1.5.0 combo or will this happen in a follow-up?

Reply:
nothing. there's just not much different from current 'main'